irc.py

```diff
@@ -0,0 +1,60 @@
+import socket
+from time import sleep
+
+
+irc = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+def connect_to(server,port):
+    irc.connect((server, port))
+    print(f"Connected to server {server}")
+
+def join_channel(channel, bot_nick):
+    irc.send(bytes('USER ' + bot_nick + " " + bot_nick + " " + bot_nick + " " + bot_nick + ' \n', 'utf-8'))
+    irc.send(bytes('NICK ' + bot_nick + ' \n', 'utf-8'))
+    irc.send(bytes('JOIN ' + channel + ' \n', 'utf-8'))
+    print(f"Joined channel {channel}")
+
+def karmas(name, point):
+    if name not in karma.keys():
+        karma[name] = 1
+    elif point == "+":
+        karma[name] += 1
+    elif point == "-":
+        karma[name] -= 1
+    irc.send(bytes("PRIVMSG " + channel + " :" + name + " Has " + str(karma[name]) + " Points. \n", 'utf-8'))
+    
+server = 'chat.freenode.net'
+port = 6667
+channel = '#pythonpune'
+bot_nick = 'gitircbot_test'
```

Comment:
Thank you for the pull request. Few changes, Since above highlighted part can be used by any other channels. So better we can move it to config file. Take a look at configparser.

Reply:
sure will update the script soon :)